visier/connector/sessions.py

```diff
@@ -212,7 +214,7 @@ def get_client_auth():
             body["redirect_uri"] = auth.redirect_uri
         response = requests.post(url=url,
                                  data=body,
-                                 headers={"apikey": auth.api_key},
+                                 headers={"apikey": auth.api_key, USER_AGENT: USER_AGENT_VALUE},
```

Comment:
We need this one to be specified separately, because no `Session` is created yet at this point.

Reply:
Yep, this is a good call.